src/backend/distributed/shared_library_init.c

```diff
@@ -356,6 +356,16 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_router_planner",
```

Comment:
```enable_router_executor``` infers enabling/disabling router executor too. This change so far only deals with planner part. Do we want to control executor too ?